src/webui/www/private/views/confirmdeletion.html

```diff
@@ -69,7 +69,7 @@
         });
 
         cancelButton.focus();
-        cancelButton.addEventListener("click", (e) => { window.qBittorrent.Client.closeWindows(); });
+        cancelButton.addEventListener("click", (e) => { window.qBittorrent.Client.closeWindow("confirmDeletionPage"); });
```

Comment:
IMO it would be better to let the caller to locate the element:
```suggestion
        cancelButton.addEventListener("click", (e) => {
            const element = document.getElementById("confirmDeletionPage");
            window.qBittorrent.Client.closeWindow(element);
        });
```

And change `window.qBittorrent.Client.closeWindow` to:
```js
    const closeWindow = (element) => {
        if (!element)  // maybe this check can be omitted if `MochaUI.closeWindow` can handle invalid element
            return;
        MochaUI.closeWindow(element);
    };
```

Just asking for opinion, I don't insist it be done in this PR.

Reply:
I don't personally have a preference between the two.

Reply:
>IMO it would be better to let the caller to locate the element:

Besides the opportunity for simpler `closeWindow()` implementation, it looks better to pass an element rather than a string for lookup.

Anyway, I can handle it in a later PR.